src/app/components/wiki-markdown/WikiMarkdown.tsx

```
@@ -12,18 +13,110 @@
+                        if (process.env.SKIP_HIGH_LEVEL_DIAGNOSTIC_LOGS !== 'true') console.warn(`WikiMarkdownLink: 'papyrus-index:' protocol with pathname '${url.pathname}' has an unknown second part after the script name: '${pathnameVariable2}'.`);
+                        return <a href={url.pathname}>{children}</a>;
+                }
+
+            }
+
+            default:
+                throw new  Error(`WikiMarkdownLink: 'papyrus-index:' protocol with pathname '${url.pathname}' does not have a valid first part after the game name.`);
+        }
+
+        // eslint-disable-next-line no-unreachable -- leaving this here in case we change things up in the future and this is no longer unreachable.
+        throw new Error(`WikiMarkdownLink: 'papyrus-index:' could not be resolved to a link: ${rawHref}`);
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)